Changes in module 'nf-core/stardist'
--- modules/nf-core/stardist/main.nf
+++ modules/nf-core/stardist/main.nf
@@ -1,7 +1,7 @@
 process STARDIST {
     tag "$meta.id"
-    label 'process_medium'
-    label 'process_gpu'
+    label 'process_high'
+    label 'process_gpu_single'

     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
